charts/vela-core/templates/defwithtemplate/cpuhpa.yaml

```
@@ -38,8 +38,14 @@ spec:
         	}
         }
         parameter: {
-        	min:     *1 | int
-        	max:     *10 | int
+        
+        	// +usage=specify the min number of replicas to which the autoscaler can scale down
+        	min: *1 | int
+        
+        	// +usage=specify the max number of of replicas to which the autoscaler can scale up
+        	max: *10 | int
+        
+        	// +usage=specify the average cpu utilization
```

```suggestion
        	// +usage=specify the average cpu utilization, for example, 50 means 50%
```

Developers often get confused by the meaning of usage, so an example can help:)

---

got it